pkg/controller/options/options.go

```diff
@@ -26,6 +26,7 @@ import (
 // use to determine if they should run at all, or store them and influence their
 // reconciliation loops.
 type AddOptions struct {
+	WhiskerEnabled      bool
```

Comment:
```suggestion
	WhiskerCRDExists      bool
```

"Enabled" seems to suggest that the CR is created, not simply that the API is present.